spec/filters/unsupported/openstruct.rb

```diff
@@ -0,0 +1,7 @@
+opal_filter "openstruct" do
```

Comment:
You're right, I didn't notice that they weren't all related.

The private/protected ones should go to `privacy.rb`.

The marshal related ones should go to `marshal.rb`, create the file since it's not there, unless the marshal related specs are in their own spec files, then just exclude it from `rubyspecs`.